telluric/util/raster_utils.py

```diff
@@ -0,0 +1,65 @@
+import os
+import math
+import json
+import rasterio
+import numpy as np
+from rasterio import shutil as sh
+from rasterio.enums import Resampling, MaskFlags
+from tempfile import TemporaryDirectory
+
+
+def _calc_overviews_factors(one, blocksize=256):
+    res = max(one.width, one.height)
+    f = math.floor(math.log2(res / blocksize))
+    factors = [2**i for i in range(1, f + 2)]
+    return factors
+
+
+def _mask_from_masked_array(data):
+    mask = data.mask[0].copy()
```

Comment:
add tests and explanation about this method

Reply:
done